.github/workflows/draft-release.yml

```diff
@@ -19,3 +19,5 @@ jobs:
     uses: 'google-github-actions/.github/.github/workflows/draft-release.yml@v0'
     with:
       version_strategy: '${{ github.event.inputs.version_strategy }}'
+    secrets:
```

Comment:
Can you add a comment explaining why the heck we have to do this?

Reply:
done